package.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/228148426-bef4ebf6-02af-4b1a-bfd9-1a02a2930e53.png)

**[OPTIONAL]**

- [ ] It would be great if you make your commit messages capitalize and meaningful to make your pull request professional 👍